angular-client/components/signup.js

```diff
@@ -1,20 +1,18 @@
 var app = angular.module('meepo')
-
-app.component('signup', {
+.component('signup', {
```

Comment:
While you can certainly get away with doing this, it's a bit risky. Makes a bit more sense to keep the `app` in there so that you can separate the lines.